scalalib/src/PublishModule.scala

```diff
@@ -18,6 +18,11 @@ trait PublishModule extends JavaModule { outer =>
   def pomSettings: T[PomSettings]
   def publishVersion: T[String]
 
+  /**
+   * Version scheme used for the model: Supported values are VersionScheme.EarlySemVer, VersionScheme.PVP, and VersionScheme.SemVerSpec
```

Comment:
Comment taken and adapted from Sbt. They don't state all the possible values, probably because these are the recommended ones.

Reply:
As far as I know, these are the three supported versioning schemes, currently.